packages/rspack/tsconfig.json

```diff
@@ -1,12 +1,13 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    // TODO: Remove skipLibCheck when fix upstream type bug
+    "skipLibCheck": true,
     "outDir": "dist",
-    "rootDir": "src",
-    "allowJs": true,
-    "strict": true
```

Comment:
I had noticed `strict` had been append to `tsconifig.base.json`, so this delete is expected.